# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/wasi-libcxx/hotfix-py3.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./libcxx/include/CMakeLists.txt.vanilla	2024-11-07 15:32:14.896795893 +0100
+++ ./libcxx/include/CMakeLists.txt	2024-11-07 15:33:37.023464032 +0100
@@ -1036,7 +1036,7 @@
 # Generate the IWYU mapping. This depends on all header files but it's also considered as an
 # "include" for dependency tracking.
 add_custom_command(OUTPUT "${LIBCXX_GENERATED_INCLUDE_DIR}/libcxx.imp"
-  COMMAND "${Python3_EXECUTABLE}" "${LIBCXX_SOURCE_DIR}/utils/generate_iwyu_mapping.py" "-o" "${LIBCXX_GENERATED_INCLUDE_DIR}/libcxx.imp"
+  COMMAND "python" "${LIBCXX_SOURCE_DIR}/utils/generate_iwyu_mapping.py" "-o" "${LIBCXX_GENERATED_INCLUDE_DIR}/libcxx.imp"
   DEPENDS ${_all_includes}
   COMMENT "Generate the mapping file for include-what-you-use"
 )
